src/ServiceInsight/Framework/UI/ScreenManager/WindowManagerEx.cs

```diff
@@ -91,6 +102,12 @@ public MessageBoxResult ShowMessageBox(string message, string caption = "", Mess
             return ShowDialog(model, null, settings);
         }
 
+        static void ApplyExitOnEscape(Window window)
+        {
+            var escapeBinding = new KeyBinding(new CloseWindowCommand(window), new KeyGesture(Key.Escape));
+            window.InputBindings.Add(escapeBinding);
+        }
```

Comment:
Can we inline this method? It seems overkill to have these two lines in a private method with only one caller.

I'd also consider inlining the `escapeBinding` variable.

Reply:
Updated the PR and incorporated the feedbacks